GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
```

Comment:
I think it is a good idea to start by introducing what the guide is going to go through. Something like
"To introduce the main features of the NeXus Constructor we will create a NeXus file template describing a simple neutron beamline, comprising a neutron source, sample and detector."